client/state/email-forwarding/reducer.js

```diff
@@ -84,11 +84,33 @@ const changeMailBoxTemporary = temporary => ( forwards, { mailbox } ) => {
 	} );
 };
 
-export const forwards = createReducer(
+export const typeReducer = createReducer(
 	null,
 	{
 		[ EMAIL_FORWARDING_REQUEST ]: () => null,
-		[ EMAIL_FORWARDING_REQUEST_SUCCESS ]: ( state, { forwards: newForwards } ) => newForwards,
+		[ EMAIL_FORWARDING_REQUEST_FAILURE ]: () => null,
+		[ EMAIL_FORWARDING_REQUEST_SUCCESS ]: ( state, { response: { type } } ) => type || null,
+	},
+	typeSchema
+);
+
+export const mxServers = createReducer(
```

Comment:
Shouldn't we name it `mxServersReducer` instead?

Reply:
I suppose it is a matter of consistency. The reason why `typeReducer` has the `Reducer` suffix is to prevent the need to rename the type variable that appears in one of the functions within it.

However, all that is pretty opaque, so consistency is probably the way to go.